vignettes/bds_exposure.Rmd

```diff
@@ -125,16 +133,21 @@ Example calls:
 adex <- derive_vars_dt(adex, new_vars_prefix = "AST", dtc = EXSTDTC)
 adex <- derive_vars_dt(adex, new_vars_prefix = "AEN", dtc = EXENDTC)
 
-head(select(adex, EXSTDTC, EXENDTC, ASTDT, AENDT))
 ```
 
-```{r eval=TRUE, echo=TRUE}
+```{r, eval=TRUE, echo=FALSE}
+dataset_vignette(adex, 
+                 display_vars = vars(USUBJID, VISIT, EXSTDTC , EXENDTC ,ASTDT, AENDT))
+```
+
+
+```{r eval=TRUE, echo=FALSE}
 adex <- select(adex, -ASTDTF, -AENDTF)
 ```
 
 
 This next examples demonstrates the datetime imputation features available
```

Comment:
I think "This next examples demonstrates" is not correct. What about "The next example demonstrates"?

I also wonder if there should be a comma before "where", i.e., 'in the `derive_vars_dtm()` function, where the time is imputed as "00:00:00"'.

Reply:
ok, updated